configs/prometheus/config.yml

```diff
@@ -1,277 +1,133 @@
-        env: org
-    - targets: ['prometheus:9090']
-      labels:
-        app: prometheus
-        service: metrics
-        env: org
-    - targets: ['alertmanager:9093']
-      labels:
-        app: alertmanager
-        service: metrics
-        env: org
-    - targets: ['elasticsearch_exporter:9114']
-      labels:
-        app: elasticsearch
-        service: metrics
-        env: org
-    - targets: ['blackbox_exporter:9115']
-      labels:
-        app: blackbox_exporter
-        service: metrics
-        env: org
-
-  #--------#
-  # PROBES #
-  #--------#
-  - job_name: blackbox-net
-    metrics_path: /probe
-    # avoid being too greedy, it's staging…
-    scrape_interval: 180s
-    params:
-      module: [http_probe]
-    static_configs:
-      - targets: ['https://world.openfoodfacts.net']
+      - targets: ["10.1.0.203:8280"]
         labels:
-          app: "off"
-          service: api
-          env: net
-      - targets: ['https://world.openfoodfacts.net/images/products/376/002/924/8001/1.100.jpg']
+          env: monitoring
+      - targets: ["10.1.0.201:8280"]
         labels:
-          app: 'off'
-          service: web
-          env: net
-      - targets: ['https://robotoff.openfoodfacts.net/api/v1/insights?count=1']
+          env: org
+      - targets: ["10.1.0.200:8280"]
         labels:
-          app: robotoff
-          service: api
           env: net
-    relabel_configs:
-      - source_labels: [__address__]
-        target_label: __param_target
-      - source_labels: [__param_target]
-        target_label: instance
-      - target_label: __address__
-        replacement: blackbox_exporter:9115
 
-  - job_name: blackbox-org
-    metrics_path: /probe
-    # avoid being too greedy
-    scrape_interval: 60s
-    params:
-      module: [http_probe]
+  - job_name: docker-node-exporter
     static_configs:
-      - targets: ['https://world.openfoodfacts.org']
+      - targets: ["10.1.0.203:8281"]
         labels:
-          app: 'off'
-          service: api
-          env: org
-      # access to statics
-      - targets: ['https://world.openfoodfacts.org/images/products/376/002/924/8001/1.100.jpg']
+          env: monitoring
+      - targets: ["10.1.0.201:8281"]
         labels:
-          app: 'off'
-          service: web
           env: org
-      - targets: ['https://world.openfoodfacts.org/']
+      - targets: ["10.1.0.200:8281"]
         labels:
-          app: 'off'
-          service: api
-          env: org
-      - targets: ['https://world.openbeautyfacts.org']
+          env: net
+
+  # TODO: use Docker Service Discovery https://prometheus.io/docs/prometheus/latest/configuration/configuration/#docker_sd_config
+  - job_name: docker-containers
+    sample_limit: 50000
+    static_configs:
+      - targets: ["prometheus:9090"]
         labels:
-          app: obf
-          service: api
-          env: org
-      - targets: ['https://world.openpetfoodfacts.org']
+          app: prometheus
```

Comment:
don't we want to add a env:monitoring label here (and to other monitoring tools) ?